packages/editor-ui/src/composables/useGlobalEntityCreation.ts

```diff
@@ -50,7 +50,7 @@ export const useGlobalEntityCreation = (
 
 	const menu = computed<Item[]>(() => {
 		// Community
-		if (!projectsStore.canCreateProjects) {
+		if (!projectsStore.isTeamProjectFeatureEnabled) {
```

Comment:
I thought we could delete `canCreateProjects` but it's also used in one other place. Looks good to me using the other flag now.